pkg/virtualKubelet/provider/provider.go

```diff
@@ -107,8 +107,9 @@ func NewLiqoProvider(nodeName, foreignClusterID, homeClusterID, internalIP strin
 	localRemappedPodCIDROpt := optTypes.NewNetworkingOption(optTypes.LocalRemappedPodCIDR, "")
 	virtualNodeNameOpt := optTypes.NewNetworkingOption(optTypes.VirtualNodeName, optTypes.NetworkingValue(nodeName))
 	grpcServerNameOpt := optTypes.NewNetworkingOption(optTypes.LiqoIpamServer, optTypes.NetworkingValue(ipamGRPCServer))
+	remoteClusterID := optTypes.NewNetworkingOption(optTypes.RemoteClusterID, optTypes.NetworkingValue(foreignClusterID))
 
-	forge.InitForger(mapper, remoteRemappedPodCIDROpt, localRemappedPodCIDROpt, virtualNodeNameOpt, grpcServerNameOpt)
+	forge.InitForger(mapper, remoteRemappedPodCIDROpt, localRemappedPodCIDROpt, virtualNodeNameOpt, remoteClusterID, grpcServerNameOpt)
```

Comment:
```suggestion
	forge.InitForger(mapper, remoteRemappedPodCIDROpt, localRemappedPodCIDROpt, virtualNodeNameOpt, remoteClusterIDOpt, grpcServerNameOpt)
```

For consistency with the other parameters